lock/src/test/java/org/infinispan/lock/ClusteredLockSplitBrainLowConsistencyTest.java

```diff
@@ -0,0 +1,75 @@
+package org.infinispan.lock;
+
+import static java.util.Arrays.asList;
+import static org.infinispan.functional.FunctionalTestUtils.await;
+import static org.testng.AssertJUnit.assertNotNull;
+import static org.testng.AssertJUnit.assertTrue;
+
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.lock.api.ClusteredLock;
+import org.infinispan.lock.api.ClusteredLockManager;
+import org.infinispan.lock.configuration.Reliability;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "clusteredLock.ClusteredLockSplitBrainLowConsistencyTest")
+public class ClusteredLockSplitBrainLowConsistencyTest extends BaseClusteredLockSplitBrainTest {
```

Comment:
Please rename to `BaseClusteredLockSplitBrainTest`. There are several other tests and test configuration files named based on "consistency", I suggest using `Dist` everywhere (and `Repl` for the others).

Reply:
I renamed